src/components/tooltip/spec.js

```diff
@@ -0,0 +1,150 @@
+import chai from 'chai';
+import { div, select, Selection } from 'utils/selection';
+import logger from 'utils/logger';
+
+import installFakeTimers from 'test/utils/fake-time';
+import emit from 'test/utils/fake-event';
+
+import { tooltip } from 'components/tooltip';
+import { config as dropdownConfig } from 'components/dropdown';
+
+export default () => {
+  describe('tooltip', () => {
+    let clock;
```

Comment:
The dropdown no longer animates so we can remove all code associated with the `clock` here